docs/user-guide/properties/index.md

```diff
@@ -0,0 +1,31 @@
+Designing Specifications
+========================
+
+This chapter describes several methods for thinking systematically about
+specifications.
+
+```{todo}
+This section is incomplete.  See
+ * [Thinking about properties tutorial](https://github.com/Certora/Tutorials/tree/master/06.Lesson_ThinkingProperties)
+ * [Video from AAVE workshop](https://youtu.be/QukwpzHhPL0?t=2185) ([slides](https://raw.githubusercontent.com/Certora/Tutorials/AAVE/AAVE/lecture2.pdf))
+```
+
```

Comment:
maybe also add a link to tutorial:
https://github.com/Certora/Tutorials/tree/master/3DayWorkshop#day-2----thinking-about-properties

Reply:
added